### PR TITLE
Add a debugging message to NodeAgent integration test

### PR DESCRIPTION
**Pull request by mangchiandjjoe**

### CODE REVIEW

File: security/tests/integration/nodeAgentTest/node_agent_test.go [hunk 1]
```diff
@@ -112,6 +112,8 @@ func TestNodeAgent(t *testing.T) {
 			continue
 		}
 
+		t.Logf("Local root certificate\n%v\nRemote root certificate\n%v", orgRootCert, rootPEM)
```
Comment (wattli) [inline security/tests/integration/nodeAgentTest/node_agent_test.go]: Can we log the current timestamp as well?

Comment (mangchiandjjoe) [reply security/tests/integration/nodeAgentTest/node_agent_test.go]: Done

**Comment by mangchiandjjoe** [comment]
We can use the fixed certificates with 100 years of expiration time for the integration test instead of creating new ones for each run

**Pull request merged**